CONTRIBUTING.md

```diff
@@ -1,6 +1,8 @@
 If you would like to contribute code to this project you can do so through GitHub by forking the repository and sending a pull request.
 
-Before Comcast accepts your code into the project you must sign the Comcast Contributor License Agreement ('CLA.pdf').
+Before Comcast accepts your code into the project you must sign the [Comcast Contributor License Agreement (CLA)](https://gist.github.com/ComcastOSS/a7b8933dd8e368535378cda25c92d19a).
+
+If you haven't previously signed a Comcast CLA, you'll automatically be asked to when you open a pull request. Alternatively, we can send you a PDF that you can sign and scan back to us. Please create a new GitHub issue to request a PDF version of the CLA.
```

Comment:
From reading this, I don't understand how someone submits the CLA. How does this process work?